components/calendar/CalendarEvent.tsx

```diff
@@ -1,37 +1,37 @@
-import React from "react"
-import { DataProps } from "@/components/calendar/Calendar"
+import React from "react";
+import { DataProps } from "@/components/EventSection";
 
-interface EventProps {
-  data: DataProps
-}
-
-const CalendarEvent: React.FC<EventProps> = ({ data }) => {
-  const descriptionLong = data.description_long.replace(/<(.|\n)*?>/g, "")
-  const dateTime = new Date(data.date_utc.replace(/-/g, "/"))
+const CalendarEvent: React.FC<DataProps> = ({
+  date_time,
+  title,
+  url,
+  date,
+  date_utc,
+  description_long,
+}) => {
+  const descriptionLong = description_long.replace(/<(.|\n)*?>/g, "");
+  const dateTime = new Date(date_utc.replace(/-/g, "/"));
   const normalDate =
     new Intl.DateTimeFormat("en-US", { weekday: "long" }).format(dateTime) +
     ", " +
-    data.date
+    date;
   return (
-    <div className="bg-gray flex gap-6 overflow-hidden column min-w-[30ch]">
-      <div className="event">
-        <div className="text-neutral-900 text-sm">{normalDate}</div>
-        <div>
-          <a
-            className="text-xl font-bold text-secondary-blue-500 no-underline leading-4"
-            href={data.url}
-            target="_blank"
-          >
-            {data.title}
-          </a>
-        </div>
-        <div className="text-primary-500 text-xs">{data.date_time}</div>
-        <div className="py-2">
-          {data.description_long && <p>{descriptionLong}</p>}
-        </div>
+    <div className="event">
```

Comment:
If the combo of tailwind classes gets overly long or reused, I think it is better to have a custom class that folks can reference/change, otherwise it gets REALLY annoying to look at within the html.